kfs-core/src/main/resources/edu/arizona/kfs/fp/document/workflow/AuxiliaryVoucherDocument.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0' encoding='UTF-8'?> 
+<data xmlns="ns:workflow" xmlns:fo="http://www.w3.org/1999/XSL/Format" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="ns:workflow resource:WorkflowData">
+  <documentTypes xmlns="ns:workflow/DocumentType" xsi:schemaLocation="ns:workflow/DocumentType resource:DocumentType">
+    <documentType>
+      <name>AV</name>
+      <parent>FP</parent>
+      <label>Auxiliary Voucher</label>
+      <docHandler>${application.url}/financialAuxiliaryVoucher.do?methodToCall=docHandler</docHandler>
```

Comment:
should be:
 ${kfs.url}/financialAuxiliaryVoucher.do?methodToCall=docHandler
please include help url also.